include/libtorrent/aux_/time.hpp

```diff
@@ -43,6 +43,9 @@ namespace libtorrent { namespace aux {
 	// resolution of this timer is about 100 ms.
 	TORRENT_EXTRA_EXPORT time_point time_now();
 	TORRENT_EXTRA_EXPORT time_point32 time_now32();
+
+	TORRENT_EXTRA_EXPORT std::time_t to_time_t(const time_point32 &tp);
+	TORRENT_EXTRA_EXPORT time_point32 from_time_t(std::time_t t);
```

Comment:
The names of these export macros aren't great. `EXTRA` refers to symbols that aren't part of the public API, but that need to be exported in order for unit tests to access them when building as a shared library. So, as this patch stands now, there are no unit tests accessing these functions, so they shouldn't be exported.

However, perhaps there should be a test that ensures that this conversion round-trips correctly. It would probably be somewhat flaky since it depends on the wall clock time. So some measure should be taken to limit flakiness.

Perhaps the round-trip can be run 10 times, and ensure it gets back to the same `time_t` at least 9 of the times.